sdk/healthinsights/azure-healthinsights-radiologyinsights/samples/sample_age_mismatch_inference.py

```diff
@@ -0,0 +1,160 @@
+        IMPRESSION:   
+        1. Normal pelvic sonography. Findings of testicular torsion.
+        A new US pelvis within the next 6 months is recommended.
+        These results have been discussed with Dr. Jones at 3 PM on November 5 2020."""
+
+        # Create ordered procedure
+        procedure_coding = models.Coding(
+            system="Http://hl7.org/fhir/ValueSet/cpt-all",
+            code="USPELVIS",
+            display="US PELVIS COMPLETE",
+        )
+        procedure_code = models.CodeableConcept(coding=[procedure_coding])
+        ordered_procedure = models.OrderedProcedure(description="US PELVIS COMPLETE", code=procedure_code)
+        # Create encounter
+        start = datetime.datetime(2021, 8, 28, 0, 0, 0, 0)
+        end = datetime.datetime(2021, 8, 28, 0, 0, 0, 0)
+        encounter = models.PatientEncounter(
+            id="encounter2",
+            class_property=models.EncounterClass.IN_PATIENT,
+            period=models.TimePeriod(start=start, end=end),
+        )
+        # Create patient info
+        birth_date = datetime.date(1959, 11, 11)
+        patient_info = models.PatientDetails(sex=models.PatientSex.FEMALE, birth_date=birth_date)
+        # Create author
+        author = models.DocumentAuthor(id="author2", full_name="authorName2")
+
+        create_date_time = datetime.datetime(2024, 2, 19, 0, 0, 0, 0, tzinfo=datetime.timezone.utc)
+        patient_document1 = models.PatientDocument(
+            type=models.DocumentType.NOTE,
+            clinical_type=models.ClinicalDocumentType.RADIOLOGY_REPORT,
+            id="doc2",
+            content=models.DocumentContent(source_type=models.DocumentContentSourceType.INLINE, value=doc_content1),
+            created_at=create_date_time,
+            specialty_type=models.SpecialtyType.RADIOLOGY,
+            administrative_metadata=models.DocumentAdministrativeMetadata(
+                ordered_procedures=[ordered_procedure], encounter_id="encounter2"
+            ),
+            authors=[author],
+            language="en",
+        )
+
+        # Construct patient
+        patient1 = models.PatientRecord(
+            id="patient_id2",
+            details=patient_info,
+            encounters=[encounter],
+            patient_documents=[patient_document1],
+        )
+
+        # Create a configuration
+        configuration = models.RadiologyInsightsModelConfiguration(verbose=False, include_evidence=True, locale="en-US")
+
+        # Construct the request with the patient and configuration
+        radiology_insights_data = models.RadiologyInsightsData(patients=[patient1], configuration=configuration)
+        job_data = models.RadiologyInsightsJob(job_data=radiology_insights_data)
+
+        # Health Insights Radiology Insights
+        try:
+            poller = radiology_insights_client.begin_infer_radiology_insights(
+                id=job_id,
+                resource=job_data,
+            )
+            job_response = poller.result()
```

Comment:
Nit lets use result instead of response. 
```suggestion
            inference_result = poller.result()
```

Reply:
changed